index.bs

```diff
@@ -2556,8 +2555,8 @@ a [=trigger state=] |triggerState|:
 
 1. Let |specEntry| be the [=map/entry=] for
     |source|'s [=attribution source/trigger specs=][|triggerState|'s [=trigger state/trigger data=]].
-1. Let |triggerTime| be the greatest [=moment=] that is strictly less than
-    |triggerState|'s [=trigger state/report window=]'s [=report window/end=].
+1. Let |triggerTime| be the greatest [=moment=] that is greater than or equal to
```

Comment:
Can we just set this to report window start?

Reply:
Done.